tile/targets/gpu/cm.jsonnet

```diff
@@ -19,23 +19,24 @@ local PARAMS = {
           // Define the stripe passes
```

Comment:
I'm thinking we should not touch the `cm` configuration until we have a way of running tests on the `cm` backend.

Reply:
passes put back in